app/views/restaurants/_form.html.haml

```diff
@@ -3,6 +3,8 @@
   = f.text_field :name, class: 'form-control', value: @restaurant.name
   = f.label :description
   = f.text_area :description, class: 'form-control', value: @restaurant.description
+  = f.label :category
+  = f.select :category, ['Pizza', 'Sushi', 'Italian', 'Thai', 'Fast Food', 'BBQ', 'French', 'Traditional', 'Vegan', 'Seafood', 'Texmex'], class: 'form-control'
```

Comment:
I would suggest that you add the categories as a constant in the Restaurant model and call it from here. Also we want to validate that the selected category is part of this array. It's a security thing.